filters/contains_grammatical_error/requirements.txt

```diff
@@ -0,0 +1 @@
+language-tool-python==2.5.5
```

Comment:
Hi @mrshu these are wonderful changes but this library is not permissible (i.e. with a GNU GPL license). It would be great if you could use a library which uses an Apache or MIT license.

Reply:
Ugh, this really will be a stretch as it seems like I'll have to put together an MIT-licensed wrapper from scratch for this to end up working somehow, but I get where you are coming from and will try to do something about it.

Reply:
@mrshu Is there any update on the MIT-licensed wrapper?

Reply:
@uyaseen the only one I know of sends data to the LanguageTool.org (free) API: https://github.com/Findus23/pyLanguagetool

Would that be permissible here? I am afraid I won't have time to put together a full-featured alternative within the next couple of weeks.

Reply:
@mrshu thanks! the license of `pyLanguagetool` seems friendly; @kaustubhdhole can comment on the permissibility aspect.

Reply:
Yes! An API call is totally acceptable till the time the API is reliable and you have the right checks for handling timeout and other exceptions. In fact, it maybe a better choice and would reduce the burden on the code of NL-Augmenter code (say as compared to a large model).